lib/scala/distribution-manager/src/main/scala/org/enso/distribution/config/GlobalConfig.scala

```diff
@@ -1,6 +1,6 @@
 package org.enso.distribution.config
 
-import org.enso.yaml.{YamlDecoder, YamlEncoder}
+import org.enso.scala.yaml.{YamlDecoder, YamlEncoder}
```

Comment:
I'd prefer we do not name packages with `scala` or `java` components in the namespace. This has a tendency to later cause problems with imports.

Reply:
There is already package `org.enso.yaml` in the `editions` project, which causes package name clashes when modularized. The name `org.enso.scala.yaml` is inspired by its project - `scala-yaml`.